internal/operators/ocs/consts.go

```diff
@@ -19,10 +19,10 @@ const (
 	// so per host requires (72-(8*3)-(5*3))/3 = 11 GiB RAM per host
 	MemoryGiBMinimalMode int64 = 11
 
-	ssdDrive     string            = "SSD"
-	hddDrive     string            = "HDD"
-	MinDiskSize  int64             = 5 //5GB is the min disk size for OCS
-	compactMode  ocsDeploymentMode = "Compact"
-	minimalMode  ocsDeploymentMode = "Minimal"
-	standardMode ocsDeploymentMode = "Standard"
+	ssdDrive       string            = "SSD"
+	hddDrive       string            = "HDD"
+	OCSMinDiskSize int64             = 25 //25GB is the min disk size for OCS
```

Comment:
Please rename to ocsMinDiskSize - unless we want to export it, const and names should start with lowercase characters.

Reply:
It is exported